src/approaches/llm_probe_approach.py

```diff
@@ -0,0 +1,137 @@
+"""Large language model (LLM) based policy guided bilevel planning approach.
```

Comment:
QOL thing: maybe let's rename "open_loop_llm" as "llm_open_loop" just so the prefix "llm" is consistently used for all approaches that are llm-based? (can just make an issue for this)